src/main/java/org/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,39 @@
+package org.bookstore.controller;
+
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.bookstore.dto.BookDto;
+import org.bookstore.dto.CreateBookDto;
+import org.bookstore.service.BookService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/books")
+@RequiredArgsConstructor
+public class BookController {
+
+    private final BookService bookService;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<BookDto> getById(@PathVariable Long id) {
```

Comment:
```suggestion
    public BookDto getById(@PathVariable Long id) {
```
Don’t  wrap the response in ResponseEntity